fecgov/fec-cms
# Feature/1635 social images update

## Summary

We're updating the social images across the site

- Resolves #1635 


## Impacted areas of the application
19 files have been changed that will affect nearly every type of page across the site(s). The only visible change will be inside the source of each page—no visual changes at this point. (See next steps at the bottom)

## Screenshots
Here's the code you'll be looking for:
![image](https://user-images.githubusercontent.com/26720877/67061404-361a6580-f12e-11e9-9944-9fbe4593144b.png)


## Related PRs
None.

## How to test
- The usual: pull the branch, `npm i`, `npm run build`, `./manage.py runserver`
- We want to make sure we don't break any of the pages, but
- Most of everything we're going to check is in the source code.
- Any time you load a page to verify it's working, check the source. (Chrome: cmd+option+u. Otherwise, right-click on a blank area of the page and choose View Page Source)
- Like in the screenshot
  (1) look for the `og:image` and `twitter:image` meta tags
  (2) double-check that the path starts with `https://www.fec.gov/static/img/social/`
  (3) verify that the image filename is as it should be. Valid values:
    - `fec-pen.png`: advisory opinions—archive and single pages
    - `fec-microphones.png`: commission meetings (all three types)—archive and single pages
    - `fec-microphone.png`: press releases—archive and single pages
    - `fec-seal.png`: weekly digest—archive and single pages
    - 
    - `fec-data.png`: everything else in the data section
    - `fec-help.png`: everything else in the help section
    - `fec-legal.png`: everything else in the legal section
    - 
    - `fec-logo.png`: everything else
- We need to check every possible type of page on the site, including the homepage, every type of page listed above, other page types _similar_ to those listed above, and other pages _completely unlike_ any of the ones listed.
- If you're the last approval, remove the Please review label, close the PR, and merge & close the branch.

## Next steps
- After this PR is merged, we'll need to verify that the images made it to the various environments
- And we'll need to use Facebook Debugger and Twitter Card Validator to reset the cached images for the more popular pages (homepage comes to mind immediately).


____
